stock_quant_manual_assign/__manifest__.py

```diff
@@ -7,7 +7,7 @@
 
 {
     "name": "Stock - Manual Quant Assignment",
-    "version": "12.0.1.1.1",
+    "version": "12.0.1.1.2",
```

Comment:
```suggestion
    "version": "12.0.1.1.1",
```
Please do not bump the version this time.